com.unity.shadergraph/Editor/Importers/ShaderGraphImporter.cs

```diff
@@ -108,6 +108,48 @@ static string[] GatherDependenciesFromSourceFile(string assetPath)
             }
         }
 
+        private Shader CreateShader(AssetImportContext ctx, string text, bool compileInitialVariants)
```

Comment:
CreateShader and SetupShaderTextureDefaults were just existing code that I copied out into their own methods for ease of reuse.